src/app.js

```diff
@@ -181,6 +202,7 @@ class Messenger extends EventEmitter {
       .then((session) => this.saveSession(session));
   }
 
+  // WIP
```

Comment:
Unclear what this WIP is referring to. Doesn't seem to match anything in the follow-up tasks...?